packages/itwinui-variables/src/themes/dark.scss

```diff
@@ -111,34 +111,34 @@ $opacities: (
   --iui-color-background-transparent-hover: #{hsl(0 0% 100% / 0.065)};
   // #endregion
   // #region Border variables
-  --iui-color-border-subtle: #{hex($bg-9)};
-  --iui-color-border: #{hex($bg-10)};
-  --iui-color-border-foreground: #{hex($fg-8)};
-  --iui-color-border-foreground-hover: #{hex($fg-6)};
-  --iui-color-border-disabled: #{hex($bg-9)};
+  --iui-color-border-subtle: #{hex($bg-900)};
```

Comment:
bg-9 converts to bg-800

Reply:
This is related to the explaination here https://github.com/iTwin/iTwinUI/pull/762#discussion_r1015991109

Reply:
previous mistake